### PR TITLE
Fixes #25022 - skip repos validation if subscribed using act_key

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: definitions/checks/repositories/validate.rb [hunk 1]
```diff
@@ -18,19 +18,29 @@ class Validate < ForemanMaintain::Check
...
-        unless absent_repos.empty?
-          spinner.update('Some repositories missing, calling `subscription-manager refresh`')
-          feature(:downstream).rhsm_refresh
-          absent_repos = feature(:downstream).absent_repos(@version)
-        end
-        unless absent_repos.empty?
-          fail!(
-            "Following repositories are not available on your system: #{absent_repos.join(', ')}"
-          )
+      if feature(:instance).subscribed_using_activationkey?
```
Comment (mbacovsky) [inline definitions/checks/repositories/validate.rb]: Why is this check in `:instance` and not in `:downstream` feature?

Comment (iNecas) [reply definitions/checks/repositories/validate.rb]: Yes, this could be downstream only. I'm ok with moving that

Comment (kgaikwad) [reply definitions/checks/repositories/validate.rb]: @mbacovsky, updated PR with a suggested change.

**Comment by mbacovsky** [comment]
Thanks @kgaikwad! The failing tests were caused by some Travis hiccup. I restarted the test and everything is green now.

**Pull request merged**